### ISSUE TITLE
🎯 North Star

### ISSUE THREAD

**Issue by birkjernstrom** [polarsource/polar#897]
# Polar North Star (~v0.8)
Laying the foundations for our future [v1 goals](https://blog.polar.sh/polar-v1-0-lets-fix-open-source-funding/) offering:
- Better management of sponsors/backers
- Ad-hoc funding behind issues & milestones
- Blogging/updates/newsletters designed for OSS maintainers with Substack-esque sharing for subscribers vs. backers
- Subscription (Credit Card, PayPal et all) for backers. Getting voting power & early access to content + more.
- Home: Birdseye insights & todos
- Public API: Integrate the content, subscription, backlog + voting/funding on your site or elsewhere

![IMG_1754 Large](https://github.com/polarsource/polar/assets/281715/50285c3f-ef98-4be0-9c34-fd471a083580)

Below are our milestones of how we're going to get there. 

_Note: Bugs and improving existing functionality based on continuous feedback will be prioritized. So the list below is not all-encompassing or guaranteed to be 100% up-to-date. You can always see efforts in flight through our Projects board or reviewing our milestones._

## Alpha Feedback & Updated dashboard
Since launching our alpha, we've received a lot of encouragement and fantastic feedback (keep it up!). So we're addressing the most common feature requests in our upcoming version:
- https://github.com/polarsource/polar/issues/898
- https://github.com/polarsource/polar/issues/895, i.e split funding between co-maintainers & contributors
- https://github.com/polarsource/polar/issues/900, i.e Embeddable SVG of Fundable Backlog in README

In parallel with:
- https://github.com/polarsource/polar/issues/901 
- https://github.com/polarsource/polar/issues/902 (Later. Stripe doesn't support PayPal for US platforms yet)
- https://github.com/polarsource/polar/issues/903 to support roadmap expansion
- Continuing on our https://github.com/polarsource/polar/issues/834

![IMG_1755 Large](https://github.com/polarsource/polar/assets/281715/5e2a967e-aac3-4532-98ac-6dca81be93e1)

## Milestones/Campaigns & Separate Maintainer vs. Backer dashboard
Another popular request has been funding a milestone - we love it! We're going to offer some flexibility here where "Campaigns" can be created which can easily map 1:1 to a GitHub milestone, but don't have to be. E.g supporting the ability to create a funding campaign for an event, meetup etc. Or combination of the two: Celebration meetup of launching v2 :-) See rough sketch below.

![IMG_1756 Large](https://github.com/polarsource/polar/assets/281715/a2eb6036-cd62-40d0-9c45-cdb13001dcc1)
![IMG_1757 Large](https://github.com/polarsource/polar/assets/281715/9bdc0e11-9ef8-4283-90ca-c9ab9d35b436)

We'll also create a separate dashboard for "Backers" and move "Dependencies" into it. Of course, maintainers often have dependencies too and the ability to fund such issues and initiatives is a core part of Polar (funding of transient dependencies). However, they are two separate modes and best served with their own dashboards for maintainers vs. backers. So we'll create a dedicated backer dashboard and ability to easily switch between both (for maintainers).

## Backer management
Giving maintainers better insights and tooling for managing backers. Easily embed an SVG on GitHub README which is kept up-to-date and leverage our API/Webhooks/SDKs to showcase them in docs/sites without manual work or streamline automation of additional upsells. Synchronizing GitHub sponsors to create one source of truth - regardless whether the backer is coming from GitHub or Polar.

![IMG_1760 Large](https://github.com/polarsource/polar/assets/281715/37687695-3438-45ef-bb20-19cc6e52108f)

## Subscriptions & Posts
Ability to create subscription tiers within Polar granting backers X voting power on issues, features and campaigns. Combined with powerful tools to create content for your community with embedded benefits to backers. From early access to posts before they're posted publicly to having certain sections or polls require a subscription. Our editing tools for creating this content will be designed with Open Source in mind. Offering seamless embedding of:
- Changelog 
- Issues (fundable)
- Milestones/campaigns
- Code/CLI commands and more

All will be available to backers in their Polar dashboard. Of course, you can easily integrate them into your own site(s) with our OAuth & API to fit whatever use case best serves your initiatives and communities.
![IMG_1758 Large](https://github.com/polarsource/polar/assets/281715/29e6acc9-d880-4656-abc5-6c1dd455914f)
![IMG_1759 Large](https://github.com/polarsource/polar/assets/281715/bcd899d9-fa65-4827-9010-04eeefb0add5)

## Onward
From there we'll continue to streamline common benefits that open source maintainers tend to upsell, e.g support (messaging), premium packages, repo invites and more. Ensuring you can easily craft these subscriptions and that they're delightful and streamlined for backers without patch work and manual coordination. Let us now in our Discussions what you want us to support and we'll get there :-)

<sub>[POL-52](https://linear.app/polar-sh/issue/POL-52)</sub>

## Upvote & Fund

- We're using [Polar.sh](https://polar.sh/polarsource) so you can upvote and help fund this issue.
- We receive the funding once the issue is completed & confirmed by you.
- Thank you in advance for helping prioritize & fund our backlog :^)

<a href="https://polar.sh/polarsource/polar/issues/897">
<picture>
  <source media="(prefers-color-scheme: dark)" srcset="https://polar.sh/api/github/polarsource/polar/issues/897/pledge.svg?darkmode=1">
  <img alt="Fund with Polar" src="https://polar.sh/api/github/polarsource/polar/issues/897/pledge.svg">
</picture>
</a>

**Comment by JacobCoffee** [comment]
Best sketches 2023 🥲

**Comment by vcarl** [comment]
Are you familiar with [OwnCo](https://www.ownco.org/)? Sounds like there are some overlapping goals there
> Co-ops could use Ownco tokens as a more versatile way to distribute ownership, but not all Ownco implementations would look like a co-op. 
> The Ownco model is focused on sharing assets based on contribution, rather than on governance.

**Comment by birkjernstrom** [comment]
Thanks for sharing @vcarl 🙏🏼

**Comment by birkjernstrom** [comment]
Excited to close this since we shipped the next generation of Polar on January 9th 😍🚀

Our new version covers the core premise and features mentioned within this issue. For clarity, some things mentioned in this issue are not done. Below are the reasons why.
- PayPal support -> Harder than we hoped for. Stripe PayPal support is only available in Europe atm. So we would need to build a completely separate integration to make it work today. This still has high potential and something we hope to explore in the future. Moved to standalone issue https://github.com/polarsource/polar/issues/902
- Milestones -> In chatting with developers, we noticed most didn't use GitHub milestones. Those who did had milestones that spanned a really long time. So funding toward a milestone in that shape of the idea started to make less sense. We got another idea from it that might resurface if we see feedback/requests around this topic.
